src/pages/creer-un-jeu/mimique/4.tsx

```diff
@@ -78,9 +82,19 @@ const MimiqueStep4 = () => {
             </p>
           )}
           <div style={{ width: '100%', textAlign: 'right', margin: '1rem 0' }}>
-            <Button variant="outlined" color="primary" onClick={onPublish} disabled={!isValid}>
-              Publier
-            </Button>
+            {isUserObservator ? (
+              <Tooltip title="Action non autorisée" arrow>
+                <span>
+                  <Button variant="outlined" color="primary" onClick={onPublish} disabled>
+                    Publier
+                  </Button>
```

Comment:
**Nit:** For all buttons in the `isUserObservator` condition,  I would remove the `onClick` handler. It's not needed because the button is disabled.